Description: use .hh extension for Bison header file
 Previously Bison generated an .hh header and rule_bison.py changed it to .h.
 That used to work fine, but with Bison 3.7 the generated .cc file #includes
 that header (instead of duplicating it), so we need to keep the original
 extension.
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: no
Last-Update: 2020-07-28

--- a/src/3rdparty/chromium/third_party/blink/renderer/build/scripts/rule_bison.py
+++ b/src/3rdparty/chromium/third_party/blink/renderer/build/scripts/rule_bison.py
@@ -104,7 +104,7 @@ outputHFile.close()
 os.unlink(outputHTmp)
 
 # Rewrite the generated header with #include guards.
-outputH = os.path.join(outputDir, inputRoot + '.h')
+outputH = outputHTmp
 
 outputHInGen = outputH.replace('gen/', '')
 headerGuard = NameStyleConverter(outputHInGen).to_header_guard()
--- a/src/3rdparty/chromium/third_party/blink/renderer/core/BUILD.gn
+++ b/src/3rdparty/chromium/third_party/blink/renderer/core/BUILD.gn
@@ -952,7 +952,7 @@ action_foreach("make_core_generated_biso
   ]
   outputs = [
     "$blink_core_output_dir/{{source_name_part}}.cc",
-    "$blink_core_output_dir/{{source_name_part}}.h",
+    "$blink_core_output_dir/{{source_name_part}}.hh",
   ]
   args = [
     "{{source}}",
--- a/src/3rdparty/chromium/third_party/blink/renderer/core/xml/xpath_parser.cc
+++ b/src/3rdparty/chromium/third_party/blink/renderer/core/xml/xpath_parser.cc
@@ -31,7 +31,7 @@
 #include "third_party/blink/renderer/core/xml/xpath_evaluator.h"
 #include "third_party/blink/renderer/core/xml/xpath_ns_resolver.h"
 #include "third_party/blink/renderer/core/xml/xpath_path.h"
-#include "third_party/blink/renderer/core/xpath_grammar.h"
+#include "third_party/blink/renderer/core/xpath_grammar.hh"
 #include "third_party/blink/renderer/platform/bindings/exception_state.h"
 #include "third_party/blink/renderer/platform/wtf/std_lib_extras.h"
 #include "third_party/blink/renderer/platform/wtf/text/string_hash.h"
